### PR TITLE
refactor(module): improve messages

### PR DESCRIPTION
**Pull request by shinGangan**
### ❓ Type of change

- [ ] 📖 Documentation (updates to the documentation, readme or JSdoc annotations)
- [x] 🐞 Bug fix (a non-breaking change that fixes an issue)
- [ ] 👌 Enhancement (improving an existing functionality like performance)
- [ ] ✨ New feature (a non-breaking change that adds functionality)
- [x] 🧹 Chore (updates to the build process or auxiliary tools and libraries)
- [ ] ⚠️ Breaking change (fix or feature that would cause existing functionality to change)

### 📚 Description

The current Nuxt CLI displays the following

```sh
$ $ npx nuxi@latest module search nuxt/ui                                          
✔ Found 1 nuxt module matching nuxt/ui for nuxt 3.10.3:

- Name           ui                                                                                                                                                                                              
- Homepage       https://ui.nuxt.com
- Compatibility  nuxt: ^3.0.0
- Repository     https://github.com/nuxt/ui
- Description    Fully styled and customizable components for Nuxt, powered by Headless UI and Tailwind CSS.
- Package        @nuxt/ui
→ Install        nuxt module add @nuxt/ui
```

This installation command does not work correctly when I actually try it. That is because the command is different.
I have changed the CLI message to avoid user confusion.

---

The following other corrections were made in this PR

- [x] Execution command and display message did not match
  - target: `nuxi module add`, `nuxi module search`

### 📝 Checklist

- [ ] I have linked an issue or discussion.
- [ ] I have added tests (if possible).
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

**Comment by shinGangan** [comment]
Hi @pi0 , please review when you have a time 🙏 .
(It is not a critical bug, so it is not a high priority.)

File: .eslintrc [hunk 1]
```diff
@@ -1,5 +1,11 @@
 {
   "extends": ["@nuxt/eslint-config"],
+  "ignorePatterns":[
```
Comment (pi0) [inline .eslintrc:3]: Please split it to another `chore:` PR 🙏🏼

File: package.json [hunk 2]
```diff
@@ -63,7 +63,7 @@
     "magicast": "^0.3.3",
     "mlly": "^1.6.1",
     "nuxt": "^3.10.2",
-    "nypm": "^0.3.6",
+    "nypm": "^0.3.8",
```
Comment (pi0) [inline package.json:66]: Please avoid unrelated changes in PR (also for lockfile)

Comment (shinGangan) [reply package.json]: Sorry, reverted this commit: d619e52290bb43c570579b91c1a7296fc85108d4

File: .eslintrc (hunk 1 above)
Comment (shinGangan) [reply .eslintrc]: Sorry, reverted this commit: 73ae4eacdf70b900708bef38de1b44c7820d2c97

This content would like to requested for review in a separate PR.

**Review by pi0 — APPROVED** [review]
Thanks!
